### PR TITLE
G4A1 - Add missing FDCAN2, rename USB peripheral

### PR DESCRIPTION
**Pull request by usbalbin**
Similar to #915

### CODE REVIEW

File: devices/stm32g4a1.yaml [hunk 1]
```diff
@@ -1,5 +1,25 @@
 _svd: ../svd/stm32g4a1.svd
 
+_modify:
+  _peripherals:
+    USB_FS_DEVICE:
+      name: USB
```
Comment (usbalbin) [inline devices/stm32g4a1.yaml:6]: Not sure this did anything...

Comment (usbalbin) [reply devices/stm32g4a1.yaml:6]: It did indeed not

File: devices/stm32g4a1.yaml [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+  USB_FS_device:
+    name: USB
+
+_derive:
+  FDCAN2:
+    _from: FDCAN
+    baseAddress: 0x40006800
+
+FDCAN2:
+  add:
```
Comment (burrbull) [inline devices/stm32g4a1.yaml:13]:
```suggestion
  _add:
```

**Review by burrbull — APPROVED** [review]

**Comment by usbalbin** [comment]
I just had a closer look at `devices/stm32g491.yaml` and  `devices/stm32g4a1.yaml` they seem to have diverged quite a bit. However looking in the [RM](https://www.st.com/resource/en/reference_manual/rm0440-stm32g4-series-advanced-armbased-32bit-mcus-stmicroelectronics.pdf#page=75) they are even in the same column under available peripherals so should be pretty much identical, I think, with AES being the only difference.

**Comment by usbalbin** [comment]
For reference:
```bash
diff devices/stm32g491.yaml devices/stm32g4a1.yaml -w --color
```

```diff
1c1,5
< _svd: ../svd/stm32g491.svd
---
> _svd: ../svd/stm32g4a1.svd
>
> _modify:
>   USB_FS_device:
>     name: USB
45,51d48
< UCPD1:
<   _include:
<     - common_patches/ucpd/rxordseten_split.yaml
<     - common_patches/ucpd/cfgr_missing_r.yaml
<     - common_patches/ucpd/tx_rx_missing_r.yaml
<     - ../peripherals/ucpd/ucpd_v1.yaml
<
96,99d92
<  - ./common_patches/usb/rename_USB_FS_peripheral_to_USB.yaml
<  - ./common_patches/usb/g4.yaml
<  - ../peripherals/rcc/rcc_g4.yaml
<  - common_patches/tim/common.yaml
100a94
>  - common_patches/tim/common.yaml
103d96
<  - ../peripherals/flash/flash_g4.yaml
```

**Pull request merged**